webdriver/tests/bidi/script/evaluate/evaluate.py

```diff
@@ -6,35 +6,53 @@
 
 
 @pytest.mark.asyncio
-async def test_eval(bidi_session, top_context):
+@pytest.mark.parametrize(
+    "expression, expected",
+    [
+        ("undefined", {"type": "undefined"}),
+        ("null", {"type": "null"}),
+        ("'foo'+ 'bar'", {"type": "string", "value": "foobar"}),
```

Comment:
I'd keep is simple. Like `'foobar'` instead of `'foo' + 'bar'`.

Reply:
fixed ✅